protos/bottle/fulfillment/v1/line_group.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineGroup {
+  string id = 1;
+  string name = 2;
+  int32 quantity = 3;
+
+  bottle.catalog.v1.Product catalog_product = 4;
+
+  enum LineGroupCategory {
+    LINE_GROUP_CATEGORY_UNSPECIFIED = 0;
+    LINE_GROUP_CATEGORY_DESKTOP = 1;
+    LINE_GROUP_CATEGORY_LAPTOP = 2;
+    LINE_GROUP_CATEGORY_SERVER = 3;
+    LINE_GROUP_CATEGORY_APPAREL = 4;
+  }
+  LineGroupCategory category = 5;
+
+  bool rush = 6;
+  double rush_price = 7;
```

Comment:
I decided to make `rush` a bool instead of relying on `rush_price != 0`. This seems more future proof if we wanted to send something to rush without charging.